### ISSUE TITLE
How can I avoid this kind of conflict?

### ISSUE THREAD

**Issue by MassMessage** [khang-nd/7.css#24]
First of all, thanks for such awesome library! my issue is I'd like to use this library to style a specific (React) component but it's changing everything else in the application, including the main page's default scrollbar and the tab component I'm using. for example the tab look like this:

![image](https://user-images.githubusercontent.com/11813549/167682726-aca2a46b-ed72-44bf-bdc6-95b3c10b4cf4.png)

instead of this:

![image](https://user-images.githubusercontent.com/11813549/167683117-984ea97e-8886-43c2-bb64-d79dadf041ba.png)

the code to generate this tab look like [this](https://github.com/Zertz/react-headless-tabs/blob/main/packages/docs/examples/Browser.tsx).  I believe most part of conflict could be solved using unique name to classes? maybe adding the library name prefix or something...

**Comment by khang-nd** [comment]
Hi @MassMessage, this framework does not rely much on classes, but it instead styles your HTML elements directly using semantic tags & accessibility attributes. Therefore it **may not coexist well** with any other styling systems.

In your use case, if you just want this framework to style only certain components, you may want to find a suitable approach to **scope** it instead of importing this framework globally.

**Comment by khang-nd** [comment]
After a few hours of digging, I would actually take back my earlier suggestion, there's **no way to scope it** (the only way that seems viable is to use this [react-scoped-css](https://github.com/gaoxiaoliangz/react-scoped-css) but it is by no means easy to set up, so let's just ditch that).

I will try to figure out a way to provide an option to scope it, may probably end up using a class prefix as you suggested or something similar.

**Comment by MassMessage** [comment]
> After a few hours of digging, I would actually take back my earlier suggestion, there's **no way to scope it** (the only way that seems viable is to use this [react-scoped-css](https://github.com/gaoxiaoliangz/react-scoped-css) but it is by no means easy to set up, so let's just ditch that).
> 
> I will try to figure out a way to provide an option to scope it, may probably end up using a class prefix as you suggested or something similar.

It seems there's a workaround with scss... something like this:

```
@use "sass:meta";

.container {
    @include meta.load-css("./baa");
}
```
it does scope the rules in the baa file. I couldn't make it work with 7 framework yet (i'm getting a include error(gonna open an issue in their repo), it's searching for the 7.css's resoruce fiels, such as button-down.svg in the `src` dir instead of  7.css's dir; but copying all the files from 7.css's dir to src folder seem to work, at least I got the expected behavior as in my initial topic). I'm not sure tho if there are plans to keep `meta.load-css` in scss tho as `@import (url)` is getting depreciated.

**Comment by khang-nd** [comment]
Ah yep, that could also be a workaround, wasn't sure if you were willing to dig into the SCSS approach so I didn't mention it, but still, the problem with icon path is there if you use this.

**Comment by MassMessage** [comment]
> Ah yep, that could also be a workaround, wasn't sure if you were willing to dig into the SCSS approach so I didn't mention it, but still, the problem with icon path is there if you use this.

This approach with scss was the only one I could find lol I didn't it would be so hard... even style-components, form what i saw, didn't solve it. Do you have any idea how solve this include issue tho? so far, no luck with that

**Comment by MassMessage** [comment]
**edit:** to anyone with same issue, this load path algorithm just pass the path to webpack's resolve so setting `resolve.modules` does solve the issue.

**Comment by khang-nd** [comment]
I published a new release (v0.6.0) that addresses this topic so that you won't have to go through so much pain to config or setup. Instructions can be found on the [documentation page](https://khang-nd.github.io/7.css/), a React example can be found [here](https://codesandbox.io/s/scoped-7-css-v91j4w).

**Comment by MassMessage** [comment]
> I published a new release (v0.6.0) that addresses this topic so that you won't have to go through so much pain to config or setup. Instructions can be found on the [documentation page](https://khang-nd.github.io/7.css/), a React example can be found [here](https://codesandbox.io/s/scoped-7-css-v91j4w).

thank you very much!! :D I feel like the setup was getting a bit hacky and messed, that helped to clean up things. I'm using the v0.6.0 already

**Comment by khang-nd** [comment]
No problem, happy to help :)
